game/scripts/npc/items/custom/item_giant_form.txt

```diff
@@ -48,7 +48,7 @@
     "AbilityTextureName"                                  "custom/giant_form_1"
     "AbilityCastPoint"                                    "0.0"
     "AbilityManaCost"                                     "50"
-    "AbilityCooldown"                                     "6.0"
+    "AbilityCooldown"                                     "4.0"
```

Comment:
Don't you have to write it like "4.0 2.0" Im not 100% sure though. Ask a real dev.